backend/compact-connect/stacks/api_stack/v1_api/api_model.py

```diff
@@ -270,6 +270,48 @@ def post_purchase_privileges_request_model(self) -> Model:
         )
         return self.api._v1_post_purchase_privileges_request_model
 
+    @property
+    def post_credentials_payment_processor_request_model(self) -> Model:
+        """Return the post payment processor credentials request model, which should only be created once per API"""
+        if hasattr(self.api, '_v1_post_credentials_payment_processor_request_model'):
+            return self.api._v1_post_credentials_payment_processor_request_model
+        self.api._v1_post_credentials_payment_processor_request_model = self.api.add_model(
+            'V1PostCredentialsPaymentProcessorRequestModel',
+            description='Post payment processor credentials request model',
+            schema=JsonSchema(
+                type=JsonSchemaType.OBJECT,
+                additional_properties=False,
+                required=['processor', 'apiLoginId', 'transactionKey'],
+                properties={
+                    'query': JsonSchema(
```

Comment:
I don't think you mean to have this `query` as a property in here.

Reply:
good catch... copy/paste error